draco/tests/renderer/altair/test_altair_renderer.py

```diff
@@ -875,7 +875,7 @@ def test_facets(spec: dict, expected_vl: dict, renderer: AltairRenderer):
                 "x": {
                     "field": "condition",
                     "type": "ordinal",
-                    "scale": {"type": "ordinal"},
+                    "scale": {},
```

Comment:
Do we need to generate an empty object here? Would be nice to have cleaner outputs.

Reply:
Fixed in 09de654